src/main/java/be/syntra/devshop/DevshopFront/services/CartService.java

```diff
@@ -11,7 +11,9 @@
 
     CartDto getCart();
 
-    void addToCart(Product product);
+    //void addToCart(Product product);
```

Comment:
commentary strikes again, haha. If you're sometimes not sure whether to remove something, I just use ctrl-X (IntelliJ has a paste-with-history keycombo -> ctrl-shift-V), or just remove it.  Or rightmouse-click -> local history -> show history. Or, würst case scenario: ctrl-Z goes a long way back too.

Reply:
removed :-) , the local history is the thing that i need i think, i tend to comment out code when doing refactorings